chapters/es/slides/chapter3_02_custom-pipeline-components.md

```diff
@@ -120,7 +120,7 @@ Cuando imprimimos en pantalla los nombres de los componentes el nuevo aparece al
 
 ```python
 # Crea el objeto nlp
-nlp = spacy.load("en_core_web_sm")
+nlp = spacy.load("es_core_news_sm")
 
 # Define un componente personalizado
 def custom_component(doc):
```

Comment:
We might as well translate `"Doc length:"` to Spanish here and below in the output? (I think there's another follow-up example below as well.)

Reply:
Done!